src/main/java/org/amahi/anywhere/adapter/ServerFilesAdapter.java

```diff
@@ -104,8 +109,28 @@ private View newView(ViewGroup container) {
 	private void bindView(ServerFile file, View view) {
 		ImageView fileIconView = (ImageView) view.findViewById(R.id.icon);
 		TextView fileTextView = (TextView) view.findViewById(R.id.text);
+		TextView fileSize = (TextView) view.findViewById(R.id.file_size);
+		TextView fileLastModified = (TextView) view.findViewById(R.id.last_modified);
+               LinearLayout moreInfo = (LinearLayout) view.findViewById(R.id.more_info);
 
 		fileTextView.setText(getFileName(file));
+
+               int size=getFileSize(file);
+
+               if(size==0){
+                 moreInfo.setVisibility(View.GONE);
+               }else{
+
+                 moreInfo.setVisibility(View.VISIBLE);
+
+                 double inMb=(size/1024.0)/1024.0;
+                 fileSize.setText(((int)(inMb*100))/100.0+"MB");
```

Comment:
This is a really bad way of conversion. You could use java's `String.format()` function to return only two decimal places.